dev.rdm.RDM.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "dev.rdm.RDM",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "rdm",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "modules": [
+        "pip-dependencies.json",
+        {
+            "name": "rdm",
+            "buildsystem": "simple",
+            "builddir": true,
+            "build-commands": [
+                "pushd ./3rdparty/lz4/build/cmake && cmake -DBUILD_STATIC_LIBS=true && make -j $FLATPAK_BUILDER_N_JOBS && popd",
+                "pushd ./src && lrelease rdm.pro && qmake VERSION=\"$(git describe --abbrev=0 --tags)+$(git rev-parse --short HEAD)\" && make -j $FLATPAK_BUILDER_N_JOBS && popd",
+                "cp ./bin/linux/release/rdm $FLATPAK_DEST/bin"
+            ],
+            "post-install": [
+                "install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/dev.rdm.RDM.svg",
+                "install -Dm644 dev.rdm.RDM.metainfo.xml -t /app/share/metainfo",
+                "install -Dm644 dev.rdm.RDM.desktop -t /app/share/applications"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/uglide/RedisDesktopManager",
```

Comment:
You probably want to have a look at https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
Thanks!